### DISCUSSION TITLE
Where to set option  -validation.max-label-names-per-series

### DISCUSSION THREAD

**Discussion by dpreston-wss** [grafana/mimir#6788] [Help and support]
#### Describe the issue
HTTP 500 messages asking to configure -validation.max-label-names-per-series

#### To Reproduce

Start feeding mimir a ton of metrics

#### Environment

- Infrastructure: kubnernetes (via helm chart)

#### Additional Context

I just want to know where to configure that option in the helm chart.

**Comment by charleskorn** [comment]
Configurations options like that one can be set through Mimir's YAML config file. 

[The config docs](https://grafana.com/docs/mimir/latest/references/configuration-parameters/) give the name of the equivalent YAML configuration option, and a YAML config file can be configured in the Helm chart with the `mimir.structuredConfig` value. 

Alternatively, limits like this one can be configured through [runtime configuration](https://grafana.com/docs/mimir/latest/configure/about-runtime-configuration/), which is managed with the `runtimeConfig` value in the Helm chart. The advantage of using runtime configuration is that it doesn't require restarting components to apply changes.
